tasker_tutorail/c10_request_examples/grep-dataset.tasks.yaml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+vars:
+  PRODUCT: Cereal
+
+tasks:
+  grep-product:
+    cmds:
+      - cat $DATASET_PATH | grep {{.PRODUCT}} | grep Germany
```

Comment:
please use ${DATASET_PATH} for env vars notation

Reply:
fixed